shared/src/main/scala/pl/metastack/metatime/Time.scala

```diff
@@ -92,5 +128,13 @@ object Time {
   }
 
   /** Current time */
-  def now(): Time = ???
+      def now() : Time = {
```

Comment:
I would suggest to use `System.currentTimeMillis()` and calculate `h`, `m`, `s` and `ms` from it.